### DISCUSSION TITLE
Add Deformed Coordinates to Exodus File

### DISCUSSION THREAD

**Discussion by ccarranza10** [sandialabs/seacas#558] [Q&A] [answered]
I recall when I used Sierra SM in the past that the results exodus file produced when opened in ParaView showed the deformed state for each time step automatically. I am not sure how this was achieved, and I was wondering how can I add that to functionality when creating my own exodus file.

**Comment by gsjaardema** [comment, marked as answer]
Typically, the convention is that the first `ndim` nodal variables, if the names start with a `d`,  are assumed to be the displacements of the nodes over time.  You can also tell ParaView which variables are the nodal displacements.

Typical names are "displacement", "disp", "displ", "d"  with suffices "x", "y", and "z", or "_x", "_y", "z".

**Reply by ccarranza10** [reply]
That's exactly what I was having an issue with. Thank you for the help. Does this similar approach work with displacement gradients?
